src/test/java/org/fogbowcloud/manager/core/ApplicationFacadeTest.java

```diff
@@ -12,1701 +12,2209 @@
-        Mockito.doReturn("fake-instance-id").when(networkOrder).getInstanceId();
+        Mockito.doReturn(FAKE_ORDER_ID).when(networkOrder).getId();
+        Mockito.doReturn(FAKE_INSTANCE_ID).when(networkOrder).getInstanceId();
 
         this.activeOrdersMap.put(networkOrder.getId(), networkOrder);
 
         List<String> expectedList = new ArrayList<>();
-        expectedList.add("fake-order-id");
+        expectedList.add(FAKE_ORDER_ID);
 
         ComputeOrder computeOrder = Mockito.mock(ComputeOrder.class);
         Mockito.doReturn(expectedList).when(computeOrder).getNetworksId();
 
+        // exercise
         this.application.changeNetworkOrderIdsToNetworInstanceIds(computeOrder);
 
+        // verify
         Assert.assertEquals(expectedList, computeOrder.getNetworksId());
     }
-	
-	@Test
-	public void testDeleteComputeOrder() throws Exception {
-		Mockito.doNothing().when(this.aaaController).authenticate(Mockito.anyString());
 
-		Order order = createComputeOrder();
-		OrderStateTransitioner.activateOrder(order);
+    // test case: When calling the method deleteCompute(), the Order passed per parameter must
+    // return its state to Closed.
+    @Test
+    public void testDeleteComputeOrder() throws Exception {
 
-		Mockito.doReturn(order.getFederationUser()).when(this.aaaController).getFederationUser(Mockito.anyString());
+        // set up
+        Order order = createComputeOrder();
+        OrderStateTransitioner.activateOrder(order);
 
-		Mockito.doNothing().when(this.aaaController).authorize(Mockito.any(FederationUser.class),
-				Mockito.any(Operation.class), Mockito.any(Order.class));
+        Mockito.doReturn(order.getFederationUser()).when(this.aaaController)
+                .getFederationUser(Mockito.anyString());
 
-		String federationTokenValue = "";
-		this.application.deleteCompute(order.getId(), federationTokenValue);
+        Mockito.doNothing().when(this.aaaController).authenticate(Mockito.anyString());
 
-		Assert.assertEquals(OrderState.CLOSED, order.getOrderState());
-	}
+        Mockito.doNothing().when(this.aaaController).authorize(Mockito.any(FederationUser.class),
+                Mockito.any(Operation.class), Mockito.any(Order.class));
 
-	@Test
-	public void testDeleteComputeOrderUnathenticated() throws Exception {
-		Mockito.doThrow(new UnauthenticatedUserException()).when(this.aaaController).authenticate(Mockito.anyString());
+        // exercise
+        this.application.deleteCompute(order.getId(), FEDERATION_TOKEN_VALUE);
 
-		Order order = createComputeOrder();
-		OrderStateTransitioner.activateOrder(order);
+        // verify
+        Assert.assertEquals(OrderState.CLOSED, order.getOrderState());
+    }
+
+    // test case: When try calling the deleteCompute() method without user authentication, it must
+    // throw UnauthenticatedUserException and the Order remains in the same state.
+    @Test
+    public void testDeleteComputeOrderUnathenticated() throws Exception {
```

Comment:
I think we can improve the name of this test. It seems we are testing the forward of exception

Reply:
... to me, it was hard to understand because the name has "Unathenticated". We are doing authorization, in fact.

Also, there is a typo in the name.

Reply:
... in addition to that, the method do no call authenticate. Which indicates we are missing verifies in the code.

Reply:
Modified the test method name for testDeleteComputeOrderWithoutAuthentication().